java/core/src/main/java/org/bondlib/StructBondType.java

```diff
@@ -355,17 +357,24 @@ private void deserializeValueAsBase(TaggedDeserializationContext context, TStruc
     }
 
     @Override
-    protected final TStruct deserializeValue(UntaggedDeserializationContext context) throws IOException {
+    protected final TStruct deserializeValue(
+        UntaggedDeserializationContext context,
+        TypeDef typeDef) throws IOException {
         TStruct value = this.newDefaultValue();
-        this.deserializeValue(context, value);
+        this.deserializeValue(context, typeDef, value);
         return value;
     }
 
-    private void deserializeValue(UntaggedDeserializationContext context, TStruct value) throws IOException {
+    private void deserializeValue(
+            UntaggedDeserializationContext context,
+            TypeDef typeDef,
+            TStruct value) throws IOException {
```

Comment:
Indentation problems. Please auto-format in IDE.

Reply:
Done.